models.js

```diff
@@ -79,3 +79,22 @@ exports.insertCommentByArticleId = (id, comment) => {
             return queryResult.rows[0];
         })
 }
+
+exports.updateArticleVotesById = (article_id, inc_votes) => {
+    const query = `
+    UPDATE articles
+    SET votes = votes + $1
+    WHERE article_id = $2;
+    `
+    return db.query(query, [ inc_votes, article_id ])
+    .then(() => {
```

Comment:
You wouldn't need to do this second db request if you added RETURNING * to the end of your first query